Specialization/Fury.lua

```diff
@@ -110,10 +113,13 @@ function Warrior:FurySingleTarget()
 	local gcd = fd.gcd;
 	local rage = fd.rage;
 	local rampageCost = fd.rampageCost;
+	local hasCondemn = MaxDps:FindSpellInSpellbook(FR.Condemn)
```

Comment:
This is really heavy piece of code, it is better to just use FindSpell instead

Reply:
also missing semicolon

Reply:
Is there an easier way to see if the player has Condemn? Since it's tied to a Quest rather than Level there's no way that I know of the check that Execute has been replaced with Condemn other than the above.

Reply:
Actually I believe you've fixed the Condemn part in the latest update so maybe this isn't needed anymore?
I'll remove it and tweak my fixes for Recklessness to fit.